layouts/diagnostic.html

```diff
@@ -0,0 +1,13 @@
+<h1 id="diagnostic">Diagnostic</h1>
+
+<p>The Diagnostic resources are concerned with the process of observing, testing, and diagnosing a patient. Note, once a
+  diagnosis is confirmed, it would be recorded in Condition so that it is part of the historical patient record.</p>
+
+<p>The Observation resource is primarily concerned with recording discrete values, answers, and results such as clinical
```

Comment:
@MaxPhilips this comes from our template (shared between Millennium and Soarian), and is a description of the category in general, where we always try to mention the biggest distinctions and examples even if it's not supported yet by us. It's intended to be a description of the HL7 concept. The list on the page is intended to be the listing of what's implemented.